spec/requests/api/cloud_tenants_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe 'CloudTenants API' do
+  describe 'GET /api/cloud_tenants' do
+    it 'lists all cloud tenants with an appropriate role' do
+      cloud_tenant = FactoryGirl.create(:cloud_tenant)
+      api_basic_authorize collection_action_identifier(:cloud_tenants, :read, :get)
+
+      expected = {
```

Comment:
Small nit, but I think this would read better similarly grouped with the other verification steps